README.md

```diff
@@ -93,26 +114,18 @@ Start the development server:
 yarn serve
 ```
 
-Build and view the site:
+Or, build and view the site in production mode:
```

Comment:
I'm wary of this wording, because the python http.server is not suitable for production (although I do understand why you'd want to do this).